docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+
+services:
+  postgres:
+    container_name: jiggl_postgres
+    image: postgres:alpine
+    environment:
+      POSTGRES_USER: jiggluser
+      POSTGRES_PASSWORD: jigglpassword
+      POSTGRES_DB: jiggl
+    ports:
+      - "5432:5432"
+    volumes:
+      - "${HOME}/jiggl-data/:/var/lib/postgresql/data"
```

Comment:
I assume none of this information is sensitive/secret?

Reply:
I don't _think_ so. The postgres user and password set here are just the default values set in the `.env.example`. Or do you mean the data we're pulling from toggl/jira?

Reply:
No, I just meant the values in this file. `volumes` as well not sensitive?

Reply:
Nah, that's just mapping a directory in the container to a local directory on your machine